modules/manage/pages/manage-xdcr/replicate-using-scopes-and-collections.adoc

```diff
@@ -0,0 +1,178 @@
+Note that a `filter scopes` field is provided; which permits strings to be entered, such that only those scopes whose names include matches to the strings are displayed in the list.
++
+Note the information that is displayed immediately above the list.
+This relates to the presentation of scope-names, in the list's *scope* column.
+Each scope-name is succeeded by the `>` symbol, and by a remote scope-name, which is by default assumed to be the name of the scope on the target system, to which replication will occur.
+If this assumption is correct, no action need be taken.
+However, if a remote scope to which replication is to occur has a different name from the one represented by default in the list, the remote-scope name must be changed: by left-clicking directly on the scope name, and editing the remote-scope name as appropriate.
+(Note that this requirement also applies to the representation of collection-names, as will be demonstrated in the next step of this procedure.)
++
+In the list currently presented, two scopes appear: which are the `_default` scope, and the scope `airlineScope`.
+In this example, data will be replicated from `airlineScope`.
+
+. Left-click on the list-row for `airlineScope`.
+The row expands, and appears as follows:
++
+image::manage-xdcr/xdcr-scope-row-expansion.png[,520,align=left]
++
+The expanded row displays a field whereby collections in the scope can be filtered, based on a string-match.
+It also features a *check all* checkbox, which allows all collections to be checked and thereby included in the intended replication; and an *include future collections* checkbox, which, if checked, ensures that collections added to the scope in future will automatically themselves be included in the replication.
++
+Currently, the scope contains a single collection, which is `airlineCollection`.
+This is checked by default: however, `airlineScope` itself has not yet been checked.
+Both `airlineScope` and `airlineCollection` must be specified for inclusion in the replication: note, however, that the default remote-names with which they are associated are incorrect, and must thereby be edited appropriately.
++
+Left-click on the scope named `airlineScope`:
++
+image::manage-xdcr/xdcr-edit-remote-scope-name.png[,210,align=left]
++
+Once focus is obtained, change the remote-scope name to `usAirlineBackupScope`:
++
+image::manage-xdcr/xdcr-modified-remote-scope-name.png[,225,align=left]
++
+In the same way, modify the remote-collection name associated with `airlineCollection` to `usAirlineBackupCollection`:
++
+image::manage-xdcr/xdcr-modified-remote-collection-name.png[,225,align=left]
++
+Note that the rules now appear in the *Mapping Rules* column, at the upper right of the screen:
++
+image::manage-xdcr/xdcr-mapping-rules-for collections.png[,300,align=left]
+
+. Filter the replication, to ensure that only documents whose `country` value is `United States` are replicated.
```

Comment:
One clarification for users that may be helpful: The filtering, if enabled, will take place across all collections mapping, and there is no option right now to have individual filters on a specific collection->collection mapping.

Reply:
I'll make this clear.